e2e/cypress/integration/ui/settings/01_user_mgmt.spec.ts

```diff
@@ -1,10 +1,12 @@
+/// <reference types="cypress"/>
```

Comment:
https://github.com/cypress-io/cypress/issues/1152#issuecomment-509729392 this suggests that there's an alternative way that doesn't require this to be added to every file... but I have tried neither.

Reply:
😮 good find, that solved the extra `index.d.ts` issue too!